modules/apps/commerce/commerce-shipping-engine-fixed-service/src/main/java/com/liferay/commerce/shipping/engine/fixed/internal/model/listener/CommerceShippingMethodModelListener.java

```diff
@@ -31,11 +34,21 @@
 
 	@Override
 	public void onBeforeRemove(CommerceShippingMethod commerceShippingMethod) {
-		_commerceShippingFixedOptionLocalService.
-			deleteCommerceShippingFixedOptions(
-				commerceShippingMethod.getCommerceShippingMethodId());
+		try {
+			_commerceShippingFixedOptionLocalService.
+				deleteCommerceShippingFixedOptions(
+					commerceShippingMethod.getCommerceShippingMethodId());
+		}
+		catch (PortalException portalException) {
+			if (_log.isWarnEnabled()) {
```

Comment:
Explain why is this a warning and not an error please?

Reply:
I had chosen this type of log to avoid too many error messages in the logs, because otherwise browsing the page where the channels defaults tab is present, many error messages would be generated